docs/tutorials/gcp_gpu.md

```diff
@@ -0,0 +1,182 @@
+    ```
+
+3.  Add your CUDA toolkit to the environment search path.
+    1.  Discover the directory of the CUDA toolkit that you installed.
+
+        ```shell
+        ls /usr/local
+        ```
+
+        The toolkit is the highest number that looks like the pattern
+        `cuda-XX.Y`.  The output of the command should resemble the
+        following:
+
+        ```shell
+        bin cuda cuda-11 cuda-11.4 etc games include lib man sbin share src
+        ```
+
+        In this case, the directory is `cuda-11.4`.
+    2.  Add the CUDA toolkit path to your environment. You can run the following
+        command to append the path to your `~/.bashrc` file.  Replace `[DIR]`
+        with the CUDA directory that you discovered in the previous step.
+
+        ```shell
+        echo "export PATH=/usr/local/[DIR]/bin${PATH:+:${PATH}}" >> ~/.bashrc
+        ```
+
+    3.  Run `source ~/.bashrc` to activate the new environment search path
+
+## 4. Install build tools
+
+Install the tools required to build qsim. This step might take a few minutes to
+complete.
+
+```shell
+sudo apt install cmake && sudo apt install pip && pip install pybind11
+```
+
+
+## 5. Create a GPU-enabled version of qsim
+
+1.  Clone the qsim repository.
+
+    ```shell
+    git clone https://github.com/quantumlib/qsim.git
+    ```
+
+2.  Run `cd qsim` to change your working directory to qsim.
+3.  Run `make` to compile qsim. When make detects the CUDA toolkit during
+    compilation, make builds the GPU version of qsim automatically.
+4.  Run `pip install .` to install your local version of qsimcirq.
+5.  Verify your installation.
+
+    ```shell
+    python3 -c "import qsimcirq; print(qsimcirq.qsim_gpu)"
+    ```
+
+    If the installation completed successfully, the output from the command
+    should resemble the following:
+
+    ```shell
+    <module 'qsimcirq.qsim_cuda' from '/home/user_org_com/qsim/qsimcirq/qsim_cuda.cpython-38-x86_64-linux-gnu.so'>
+    ```
+
+
+## 6. Verify your installation
+
+You can use the code below to verify that qsim uses your GPU. You can paste the
+code directly into the REPL, or paste the code in a file.
+
+```
+# Import Cirq and qsim
+import cirq
+import qsimcirq
+
+# Instantiate qubits and create a circuit
+q0, q1 = cirq.LineQubit.range(2)
+circuit = cirq.Circuit(cirq.H(q0), cirq.CX(q0, q1))
+
+# Instantiate a simulator that uses the GPU
+gpu_options = qsimcirq.QSimOptions(use_gpu=True)
+qsim_simulator = qsimcirq.QSimSimulator(qsim_options=gpu_options)
+
+# Run the simulation
+print("Running simulation for the following circuit:")
+print(circuit)
+
+qsim_results = qsim_simulator.compute_amplitudes(
+    circuit, bitstrings=[0b00, 0b01])
+
+print("qsim results:")
+print(qsim_results)
+```
+
+After a moment, you should see a result that looks similar to the following.
+
+```
+[(0.7071067690849304+0j), 0j]
+```
+
+## Next steps
+
+After you finish, don't forget to stop or delete your VM on the Compute
+Instances dashboard.
```

Comment:
This should be emphasized and include a note about billing (e.g. end with "...to prevent further billing.")

Reply:
Fixed.